resources/views/projects/tasks/create.blade.php

```diff
@@ -0,0 +1,83 @@
+@extends('layouts.app')
+
+@section('title', 'New task')
+
+@section('content')
+<div class="padded content">
+	<h1>New task</h1>
+	<!-- TODO: action url -->
+	<form class="ui form {{ $errors->any() ? 'error': '' }}" method="post" enctype="multipart/form-data" action="">
```

Comment:
Layout first, function second. Your `form` should be inside your grid. Your title as well and then you don't need to remove the padding from the column